coordinatorlib/src/test/scala/com/socrata/datacoordinator/truth/metadata/sql/PostgresDatasetMapWriterTest.scala

```diff
@@ -1,21 +1,47 @@
 package com.socrata.datacoordinator.truth.metadata.sql
 
-import org.scalatest.{BeforeAndAfterAll, FunSuite}
-import org.scalatest.MustMatchers
-import java.sql.{SQLException, Connection, DriverManager}
+import java.sql.{Connection, DriverManager, SQLException}
+
 import com.rojoma.simplearm.util._
-import com.socrata.datacoordinator.truth.metadata._
-import com.socrata.datacoordinator.util.collection.ColumnIdMap
-import com.socrata.datacoordinator.id.{RollupName, UserColumnId, RowId, ColumnId}
+import com.socrata.datacoordinator.id.{ColumnId, RollupName, UserColumnId}
+import com.socrata.datacoordinator.truth.metadata.{CopyPair, _}
+import com.socrata.datacoordinator.truth.migration.Migration
 import com.socrata.datacoordinator.util.NoopTimingReport
+import com.socrata.datacoordinator.util.collection.ColumnIdMap
+import com.socrata.soql.environment.{ColumnName, TypeName}
+import org.scalatest.{BeforeAndAfterAll, FunSuite, MustMatchers}
+
 import scala.concurrent.duration.Duration
-import com.socrata.soql.environment.TypeName
-import com.socrata.datacoordinator.truth.metadata.CopyPair
-import com.socrata.datacoordinator.truth.migration.Migration
 
+/**
+  * Tests the Postgres implementation of DatasetMapWriter
+  *   _____________________________________
+  *  / Beware! Unlike the rest of our unit \
+  *  | tests which use H2, this test runs  |
+  *  | against a test postgres database    |
+  *  \ blist_test with user blist.         /
```

Comment:
We should probably just change that to have it use a data_coordinator_test database...